doc/api/deprecations.md

```diff
@@ -2814,6 +2814,13 @@ Type: Documentation-only (supports [`--pending-deprecation`][])
 The `'hash'` and `'mgf1Hash'` options are replaced with `'hashAlgorithm'`
 and `'mgf1HashAlgorithm'`.
 
+### DEP0155: Trailing slashes in pattern specifier resolutions
+
+Type: Runtime
```

Comment:
Would it make sense to doc-only deprecate it first so we can backport the deprecation?

Reply:
When I create the backport PR separately I'll make it a doc only / pending deprecation specifically for the backports.

Reply:
I think it'd be better to have the doc-only PR land on master first.

Reply:
For the 16 version I think we need to get the full runtime deprecation out sooner rather than later - I wouldn't want to delay on that.

Reply:
Actually I guess 17 is just around the corner so that might actually work.

Reply:
Runtime deprecation are considered semver-major, so this PR probably cannot land on v16.x line: https://github.com/nodejs/node/blob/7dbc0f74d011739db0165b81cc2ac78b36c3d100/doc/guides/collaborator-guide.md#L438-L440

Reply:
Ok, I've fixed this up to be docs-only for now.